tests/test_hook_stop_cadence_p3_substantive.py

```
@@ -0,0 +1,261 @@
+
+
+def test_filler_classification_keeps_below_threshold(
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    """A filler prompt is classified False — isolates the classifier effect.
+
+    Same seeded window as ``test_substantive_above_threshold_fires`` minus
+    one true: with a substantive push the ratio would clear 0.8 threshold,
+    but a filler push leaves it just under.
+    """
+    serr, calls, state_dir = _run_stop(
```

Ruff (0.15.14)

[warning] Unpacked variable `serr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)